tests/phpunit/tests/rest-api/rest-pattern-directory-controller.php

```diff
@@ -310,6 +340,174 @@ static function( $response ) {
 		$this->assertSame( 'modified the cache', $patterns[0] );
 	}
 
+	/**
+	 * Tests if the provided query args are passed through to the wp.org API.
+	 *
+	 * @dataProvider data_get_items_query_args
+	 *
+	 * @covers WP_REST_Pattern_Directory_Controller::get_items
+	 *
+	 * @since 6.2.0
```

Comment:
Per [56070](https://core.trac.wordpress.org/ticket/56070) proposal, the document formate needs to changes for consistency. 

```suggestion
	 * @since 6.2.0
	 *
	 * @ticket 57501
	 *
	 * @covers WP_REST_Pattern_Directory_Controller::get_items
	 *
	 * @dataProvider data_get_items_query_args
```

Reply:
Noting: that proposal has not been adopted yet as it's still in discussion. That said, it's okay to go ahead with the changes here should the proposal be adopted.